cli/src/commands/runTests.js

```diff
@@ -509,7 +516,13 @@ async function runTestGroup(
           //
           // i.e. underscore/v1.x.x/test-underscore.js
           //      underscore/v1.x.x/flow-v0.22.x/test-underscore.js
-          const destBasename = idx + '-' + path.basename(filePath);
+          //
+          // Only mangles the name when there's a naming collision. Otherwise, uses the original.
+          const fileName = path.basename(filePath);
+          const destBasename = copiedFileNames.has(fileName)
+            ? `${idx}-${fileName}`
+            : fileName;
+          copiedFileNames.add(destBasename);
```

Comment:
Whatever happens with `copiedFileNames`?

Reply:
Nothing; it's used as a way to track which file names have been copied over to watch for collisions. See line 522.

Reply:
Then I'd remove it, if its not being printed and is only for debugging purposes.

Reply:
Forget all of what I said, this LGTM.

Reply:
Resolved offline, leaving it in. Salient takeaway:

<img width="237" alt="screen shot 2018-03-09 at 3 24 44 am" src="https://user-images.githubusercontent.com/5187404/37187203-87de19de-2349-11e8-9943-6540690d906c.png">